app/models/compute_resource.rb

```diff
@@ -407,7 +407,7 @@ def memory_gb_to_bytes(memory_size)
   end
 
   def to_bool(value)
-    ['1', 'true'].include?(value.to_s.downcase) unless value.nil?
```

Comment:
We should keep here original implementation as we discussed in previous comment https://github.com/theforeman/foreman/pull/9394#discussion_r963544725